### ISSUE TITLE
HTML CodeSniffer: undefined is not an object (evaluating 'window.HTMLCS.process')

### ISSUE THREAD

**Issue by joacim-boive** [pa11y/pa11y-dashboard#198]
Trying to run any test agains any page on https://www.ecster.se and I get the below regardless what test or timeouts I set.

Running it against other websites (like lensway.se) works just fine.

Any ideas?

```
 Starting to run one-off task @ Fri Nov 10 2017 13:13:27 GMT+0000 (UTC)
web_1       | Starting task 5a05a5777872a20900084a7e
web_1       |   > PhantomJS page created
web_1       |   > Opening "https://www.ecster.se" in PhantomJS
web_1       |   > Opened "https://www.ecster.se" in PhantomJS
web_1       |   > Injecting HTML CodeSniffer
web_1       |   > Injecting Pa11y
web_1       |   > Running Pa11y on the page
web_1       |   > Waiting for 10000ms
web_1       |   > Test function errored for "https://www.ecster.se"
web_1       |   > PhantomJS page closed
web_1       |   > PhantomJS browser exited
web_1       | Failed to finish task 5a05a5777872a20900084a7e: HTML CodeSniffer: undefined is not an object (evaluating 'window.HTMLCS.process')
web_1       | Finished running one-off task @ Fri Nov 10 2017 13:13:38 GMT+0000 (UTC)
```

**Comment by jasonday** [comment]
Same issue for me on both target.com and https://www.llbean.com/llb/shop/32709

I am able to scan the llbean homepage, but this page errors out. 

Installed the dashboard this AM (downloaded repo) with no changes to dependencies.

**Comment by hollsk** [comment]
https://github.com/pa11y/pa11y/issues/335

just linking these two issues - once we've resolved it in Pa11y, we'll be able to resolve it in Dashboard 😄

**Comment by rowanmanning** [comment]
This should be fixed if you update to Pa11y Dashboard 2.4.1.

Sorry for the wait, and please let us know if you're still having the problem and we can reopen
